### DISCUSSION TITLE
Windows cover with remote (Tasmota)

### DISCUSSION THREAD

**Discussion by rucanunes** [arendst/Tasmota#16265] [Q&A]
Hello,

I would like to add switches to my windows cover motors. 
I would like them to be tasmota, with remote and with percentage open or closed.
Could you please point me on the right direction?

Tks

**Comment by barbudor** [comment]
May be you could detail what you already have, how does the existing motor works and are controlled
In most of the cases, motors are already controlled by physical buttons on which you can connect in parallel relay to simulate button pressure and have those relays controlled by Tasmota

**Reply by rucanunes** [reply]
Thank you for your reply.
I have these ones controlled via Broadlink add-on....

https://a.aliexpress.com/_mqPrr4o

I have two problems with these, I don't know the position of the shutters and they are not connected via wifi.

**Reply by rucanunes** [reply]
I was thinking about replacing them with these ones and flash them with Tasmota.

https://a.aliexpress.com/_ms99JE8

**Reply by barbudor** [reply]
Do you have any confirmation that these would be flashable with Tasmota ?
Even if you can get Tasmota flashed on it, it is unsure that you could get the RF remote to work.

I suggest that you stick with those listed here: https://templates.blakadder.com/cover.html
At least someone has been successfull with those

**Reply by rucanunes** [reply]
There's where I saw these.

SC511WSC
